birdhouse/CadQuery.py

```diff
@@ -0,0 +1,42 @@
+height = 85.0
+width = 120.0
+thickness = 2.0
+diameter = 22.0
+holeDia = 28.0
+hookHeight = 10.0
+
+def turn90(shape, shouldRotate):
+    if shouldRotate:
+        return shape.rotateAboutCenter((0,0,1),90)
+    return shape
+
+def frame(shouldRotate = False):
+    temp = cq.Workplane("YZ")\
+    .moveTo(-width*0.95/2,0).lineTo(width*0.95/2,0).lineTo(0,height).close()\
+    .extrude(width).faces('|X').shell(thickness).translate((-width/2,0,0))
+    return turn90(temp, shouldRotate)
+
+def hookBody():
+    return cq.Workplane("YZ")\
+    .moveTo(-width/4,1).lineTo(0,hookHeight).lineTo(width/4,1)\
+    .lineTo(width/4+5,0).lineTo(-width/4-5,0).close()\
+    .extrude(thickness*1.5).edges("|X exc (<Y or >Y)").fillet(10).translate((-thickness*1.5/2,0,height))
```

Comment:
I did struggle a bit with the selectors on this shape, it looks like this.
![image](https://user-images.githubusercontent.com/29681384/102930227-247ddc00-44f0-11eb-87a0-8d7a56319a2e.png)
(drawn in freecad because I had it open)
I wanted to add fillets to all of the vertices apart from the bottom two, and I only seemed to be able to do the top one. Something I'm doing wrong, or is it that it only does external corners not internal?